Yoast/wordpress-seo
# Remove the address block and schema template from Free

## Context


* The `address` block and schema templates have been renamed to `job-location`, and moved to Premium.

## Summary


This PR can be summarized in the following changelog entry:

* Removes the `address` block and schema templates from Free (they are now in Premium).

## Relevant technical choices:

* The `address` block and schema templates have been renamed to `job-location`, and moved to Premium.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

**Preparation**
* In the `schema-blocks` directory in the monorepo, run `yarn build`.
* Make sure you have the monorepo linked and the feature flag turned on.
* Build Free.

**Test**
* Open the inspector on the `Elements` tab.
* Create a new post.
* Inspect the `script` tags in the `head`.
* The `yoast/address-template` block and schema template should NOT be present.

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #
